api/config/v1/flags.go

```diff
@@ -162,6 +163,8 @@ func (f *Flags) UpdateFromCLIFlags(c *cli.Context, flags []cli.Flag) {
 				updateFromCLIFlag(&f.GFD.NoTimestamp, c, n)
 			case "machine-type-file":
 				updateFromCLIFlag(&f.GFD.MachineTypeFile, c, n)
+			case "vfio-pci-driver-mode":
```

Comment:
Could you add this to the Helm templates as well

Reply:
Done